WcaOnRails/app/views/git_hub/edit_translation.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h4>Update website translation</h4>
+    </div>
+    <div class="panel-body">
+      <%= simple_form_for :translation, url: update_translation_path, method: :patch do |f| %>
+        <%= f.input :locale, collection: Locales::AVAILABLE.keys %>
```

Comment:
Note: we support updating existing translation files. That's intentional as adding a new one requires more setup anyway.